contrib/tuigreet/template.py

```diff
@@ -0,0 +1,16 @@
+pkgname = "tuigreet"
+pkgver = "0.8.0"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = [
+    "cargo",
+    "scdoc",
+]
+makedepends = ["rust-std"]
+depends = ["greetd"]
+pkgdesc = "Graphical console greeter for greetd"
```

Comment:
so is it graphical, or console? :) the name implies it's console and no graphical stuff is involved, but the description says otherwise?